packages/salesforcedx-vscode-core/test/jest/telemetry/index.test.ts

```diff
@@ -95,7 +97,7 @@ describe('Telemetry', () => {
       expect(telemetryEnabled).toEqual(true);
 
       await showTelemetryMessage(mockExtensionContext);
-      expect(globalStateTelemetrySpy).toHaveBeenCalledTimes(2);
+      expect(globalStateTelemetrySpy).toHaveBeenCalledTimes(3);
       expect(globalStateTelemetrySpy).toHaveBeenCalledWith(TELEMETRY_GLOBAL_VALUE);
```

Comment:
For consistency in the test. 
```suggestion
      expect(globalStateTelemetrySpy).toHaveBeenCalledWith(TELEMETRY_GLOBAL_USER_ID);
```